dockerfiles/README.md

```diff
@@ -0,0 +1,17 @@
+The Dockerfile(s) for the build are in this directory.
+
+* `Dockerfile` is the file used for most builds including when using localhost.
+
+* `Dockerfile.openshift` is a custom file with the FROM entries modified to point
+  to local images in the Openshift repository.
+  (Note: we may be able to combine this with `Dockerfile` again when we have access Openshift 4).
```

Comment:
```suggestion
  (Note: we may be able to combine this with `Dockerfile` again when we have access to Openshift 4).
```

Reply:
Sorry, merged before I saw this one. Documentation is harder to write than code. ;)

Reply:
Hmm, is the period supposed to be in the parenthesis? This looks correct. . . But
["When a complete, independent sentence is entirely enclosed by parentheses, the period goes inside the closing parenthesis."](https://www.grammarly.com/blog/period/)

Didn't know about that edge case!

Reply:
Yeah, I was fixing my own mistakes here. It makes sense to me from a grammar perspective. A period ends an independent clause, which in this case is entirely in the parentheses. Not a super important thing, but something I would definitely have wanted corrected when I was working as a journalist.